spec/helpers.js

```diff
@@ -167,6 +166,7 @@ describe('helper registration', () => {
             'titleize',
             'trim',
             'uppercase',
+            'ellipsis',
```

Comment:
So this will be a breaking change. Earlier if html tag was present it used to ignore/sanitize it before running `ellipsis` on the actual test